File: 2-Semestre/Taller de Programación/Personal/Python-C#/Ejercicio 7.cs
/*
Calcular e imprimir el valor de una cuota en pesos (CLP) de un crédito hipotecario expresado en UF.
    a) El valor de la UF es el valor del día.
    b) El valor del crédito debe ser ingresado por el usuario.
*/

using System;

namespace Ejercicio_7
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Este programa calcula el valor de una cuota en pesos (CLP) de un crédito hipotecario expresado en UF.\n");

            double uf, cuota;

            do
            {
                Console.Write("Ingrese el valor actual de la UF: ");
                uf = Convert.ToDouble(Console.ReadLine());

                if (uf <= 0)
                {
                    Console.WriteLine("[!] El valor de la UF no puede ser menor o igual a 0");
                }
            } while (uf <= 0);

            do
            {
                Console.Write("Ingrese el valor de la cuota en UF: ");
                cuota = Convert.ToDouble(Console.ReadLine());

                if (cuota <= 0)
                {
                    Console.WriteLine("[!] El valor de la cuota no puede ser menor o igual a 0");
                }
            } while (cuota <= 0);

            double cuota_clp = cuota * uf;
            Console.WriteLine($"\nEl valor de la cuota es de ${cuota_clp}");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Diagramas de flujo-Python (Semana 5)/Ejercicio 7.py
'''
Calcular e imprimir el valor de una cuota en pesos (CLP) de un crédito hipotecario expresado en UF.
    a) El valor de la UF es el valor del día.
    b) El valor del crédito debe ser ingresado por el usuario.
'''

print("Este programa calcula el valor de una cuota en pesos (CLP) de un crédito hipotecario expresado en UF.")
uf = float(input("Ingrese el valor actual de la UF: "))
cuota = float(input("Ingrese el valor de la cuota en UF: "))

while uf <= 0 or cuota <= 0:
    print("[!] Ingrese nuevamente")
    uf = float(input("Ingrese el valor actual de la UF: "))
    cuota = float(input("Ingrese el valor de la cuota en UF: "))

cuota_clp = cuota * uf
print(f"El valor de la cuota es de ${cuota_clp}")